lib/index.js

```diff
@@ -104,7 +103,7 @@ class PackWalker extends IgnoreWalker {
     super(options)
     this.isPackage = options.isPackage
     this.seen = options.seen || new Set()
-    this.tree = options.tree // no default, we'll load the tree later if we need to
+    this.tree = options.tree
```

Comment:
why not have the signature be `tree[, options]` since tree is now required?

Reply:
Went ahead and made that change.